pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/IndexLoadingConfigMetadata.java

```diff
@@ -32,10 +32,14 @@
   public static final String KEY_OF_LOADING_INVERTED_INDEX = "metadata.loading.inverted.index.columns";
   public static final String KEY_OF_SEGMENT_FORMAT_VERSION = "segment.format.version";
   public static final String KEY_OF_ENABLE_DEFAULT_COLUMNS = "enable.default.columns";
+  public static final String KEY_OF_STAR_TREE_FORMAT_VERSION = "startree.format.version";
+
   private final Set<String> _loadingInvertedIndexColumnSet = new HashSet<String>();
   private final String DEFAULT_SEGMENT_FORMAT = "v1";
```

Comment:
we have too many v1 and v2 etc. Lets use some thing representative. You can call JAVA_SER and OFF_HEAP or something like that